<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
<<<<<<< HEAD
    <width>400</width>
    <height>300</height>
=======
    <width>491</width>
    <height>293</height>
>>>>>>> c3bafb205848e51123afdaba602d6ba05f5d7812
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
<<<<<<< HEAD
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
=======
  <widget class="QWidget" name="centralWidget">
   <widget class="QLCDNumber" name="lcd_bestscore">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>0</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcd_score">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button_1">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>24</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_2">
    <property name="geometry">
     <rect>
      <x>395</x>
      <y>24</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>47</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>47</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>80</y>
      <width>47</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>47</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>130</y>
      <width>47</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>130</y>
      <width>47</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>130</y>
      <width>47</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>47</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>180</y>
      <width>47</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>180</y>
      <width>47</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>180</y>
      <width>47</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>47</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>230</y>
      <width>47</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>230</y>
      <width>47</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>230</y>
      <width>47</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_bestscore">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>47</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_score">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>47</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
>>>>>>> c3bafb205848e51123afdaba602d6ba05f5d7812
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
<<<<<<< HEAD
     <width>400</width>
=======
     <width>491</width>
>>>>>>> c3bafb205848e51123afdaba602d6ba05f5d7812
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
<<<<<<< HEAD
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
=======
   <property name="enabled">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
>>>>>>> c3bafb205848e51123afdaba602d6ba05f5d7812
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
<<<<<<< HEAD
  <widget class="QStatusBar" name="statusBar"/>
=======
  <widget class="QStatusBar" name="statusBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
>>>>>>> c3bafb205848e51123afdaba602d6ba05f5d7812
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
